samples/model-builder/conftest.py

```diff
@@ -364,3 +364,47 @@ def mock_endpoint_explain(mock_endpoint):
     with patch.object(mock_endpoint, "explain") as mock_endpoint_explain:
         mock_get_endpoint.return_value = mock_endpoint
         yield mock_endpoint_explain
+
+
+"""
+----------------------------------------------------------------------------
+FeatureStore Fixtures
+----------------------------------------------------------------------------
+"""
+
+
+@pytest.fixture
+def mock_featurestore():
+    mock = MagicMock(aiplatform.featurestore.Featurestore)
+    yield mock
+
+
+@pytest.fixture
+def mock_get_featurestore(mock_featurestore):
+    with patch.object(aiplatform.featurestore, "Featurestore") as mock_get_featurestore:
+        mock_get_featurestore.return_value = mock_featurestore
+        yield mock_get_featurestore
+
+
+@pytest.fixture
+def mock_create_featurestore():
+    with patch.object(aiplatform.Featurestore, "create") as mock:
+        yield mock
+
+
+@pytest.fixture
+def mock_create_entity_type():
+    with patch.object(aiplatform.EntityType, "create") as mock:
```

Comment:
Please follow the same pattern for resource creation below
```
@pytest.fixture
def mock_create_featurestore(mock_featurestore):
    with patch.object(
        aiplatform.featurestore.Featurestore, "create"
    ) as mock_create_featurestore:
        mock_create_featurestore.return_value = mock_featurestore
        yield mock_create_featurestore
```

Reply:
this need to apply to `mock_feature` as well.

Reply:
Fixed